### PR TITLE
[release/7.0] Add workload win-arm64 support

### PR DESCRIPTION
**Pull request by steveisok**

### CODE REVIEW

**Review by lewing — APPROVED** [review]

File: src/workloads/workloads.csproj [hunk 1]
```diff
@@ -111,7 +111,9 @@
     <ItemGroup>
       <!-- Set SupportsMachineArch metadata to true if a specific manifest needs to support arm64 for Visual Studio. DO NOT set if for all feature bands as
            it changes the SWIX authoring and older versions of Visual Studio will fail to install. -->
-      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-*.nupkg" MsiVersion="$(MsiVersion)"/>
+      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-%(WorkloadSdkBandVersions.Identity).*.nupkg"
+                        MsiVersion="$(MsiVersion)"
+                        SupportsMachineArch="%(SupportsMachineArch)" />
```
Comment (radical) [inline src/workloads/workloads.csproj:116]: Should the metadata reference be qualified?

File: eng/Versions.props [hunk 2]
```diff
@@ -26,6 +26,10 @@
     <MicrosoftNETWorkloadEmscriptennet6Manifest70100Version>7.0.1</MicrosoftNETWorkloadEmscriptennet6Manifest70100Version>
     <MicrosoftNETWorkloadEmscriptennet7Manifest70100Version>7.0.1</MicrosoftNETWorkloadEmscriptennet7Manifest70100Version>
   </PropertyGroup>
+  <ItemGroup>
+    <!-- The bands we want to produce workload manifests for -->
```
Comment (radical) [inline eng/Versions.props:30]:
```suggestion
    <!-- The bands we want to produce workload manifests for.
         `SupportsMachineArch`: Metadata describing whether the VS authoring supports the machineArch SWIX property.
    -->
```

I found that in

https://github.com/dotnet/arcade/blob/14468017b1f17d4f780aec3b0b3d20511fef308b/src/Microsoft.DotNet.Build.Tasks.Workloads/src/Metadata.cs#L41-L44

File: src/workloads/workloads.csproj [hunk 3]
```diff
@@ -111,7 +111,9 @@
     <ItemGroup>
       <!-- Set SupportsMachineArch metadata to true if a specific manifest needs to support arm64 for Visual Studio. DO NOT set if for all feature bands as
            it changes the SWIX authoring and older versions of Visual Studio will fail to install. -->
-      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-*.nupkg" MsiVersion="$(MsiVersion)"/>
+      <ManifestPackages Include="$(PackageSource)Microsoft.NET.Workload.Mono.ToolChain.*Manifest-%(WorkloadSdkBandVersions.Identity).*.nupkg"
```
Comment (radical) [inline src/workloads/workloads.csproj:114]: Can we validate this change in the PR itself? Maybe with a one-off run of the pipeline for this?

Comment (lewing) [reply src/workloads/workloads.csproj:114]: we're using the same pattern in emsdk and this was built in the maui arm branch previously

File: eng/Versions.props (hunk 2 above)
Comment (lewing) [reply eng/Versions.props:30]: that would be good for main

**Review by radical — APPROVED** [review]

**Pull request merged**